### PR TITLE
fix: Coherent varticle timeline in dark mode

### LINKED ISSUE #3751: Vertical timeline is not coherent in Dark Mode
### Description

I was seeing the Landscape page in dark mode and the vertical timeline is very hard to spot.

Screenshot 2023-02-13 at 3 09 12 PM

### Expected Behavior

The timeline colour must adapt with the dark mode.

### Environment:
- Host OS: 
- Browser: 

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Manishch27**
Signed-off-by: Manishch27 <manishchaudhary2705@gmail.com>

**Description**

This PR fixes #3751

This PR fixes the vertical timeline that is not coherent in the dark mode problem.

Now, in the dark mode verticle timeline is visible clearly. 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit e18fc4d8d34a3d161b3f6047e82c5c0fd0ed973b at: https://63ea8afd585a881c8b7fadc4--layer5.netlify.app

**Review by ShuvamChatterjee28 — CHANGES_REQUESTED** [review]
In light mode, it is showing white color. Please fix it.

File: src/sections/Landscape/LandscapeGrid.style.js [hunk 1]
```diff
@@ -149,7 +149,7 @@ export const LandscapePageWrapper = styled.div`
 		margin: 2.5em 0;
 	}
 	.vertical-timeline::before {
-		background: #1E2117;
+		background: #fff;
```
Comment (Nikhil-Ladha) [inline src/sections/Landscape/LandscapeGrid.style.js:152]: Please check the code on how we are switching colors based on the `theme` value, and update the code accordingly.
Use the `primaryColor` for the dark theme and `tertiaryColor` for the light theme

File: src/sections/Landscape/LandscapeGrid.style.js [hunk 2]
```diff
@@ -166,7 +166,7 @@ export const LandscapePageWrapper = styled.div`
 	.meshBubble {
 		background: #00B39F;
 		margin-top: 5rem;
-		border: 8px solid #1E2117;
+		border: 8px solid #fff;
```
Comment (Nikhil-Ladha) [inline src/sections/Landscape/LandscapeGrid.style.js:169]: Same as above.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

File: src/sections/Landscape/LandscapeGrid.style.js (hunk 1 above)
Comment (ShuvamChatterjee28) [reply src/sections/Landscape/LandscapeGrid.style.js:152]: Yes follow that what @Nikhil-Ladha said (it's much better to follow). 

Please don't change the colour of that centre bubble. It looks good on light and dark background.

**Comment by Manishch27** [comment]
@Nikhil-Ladha and @ShuvamChatterjee28, I will do changes according to you and create a pull request again.

**Comment by eeshaanSA** [comment]
@Manishch27 please wait for assignment of issue before raising a PR. Helps us keep track of issues.

**Comment by l5io** [comment]
🚀 Preview for commit ce149021461db2099c470915ff2c79fe980f2dbd at: https://63f0668438cdc3500abb53b3--layer5.netlify.app

**Comment by Nikhil-Ladha** [comment]
Please fix the DCO issue by following the steps mentioned [here](https://github.com/layer5io/layer5/pull/3756/checks?check_run_id=11399645779)

**Review by ShuvamChatterjee28 — CHANGES_REQUESTED** [review]
In the outer circle of the mesh bubble, please follow the same architecture as you have followed for the vertical timeline. It must be coherent too.

**Comment by ShuvamChatterjee28** [comment]
Also please follow the steps @Nikhil-Ladha mentioned to fix the DCO issue. Steps are [here](https://github.com/layer5io/layer5/pull/3756/checks?check_run_id=11399645779). This is important.

**Comment by Manishch27** [comment]
@ShuvamChatterjee28, I think I fixed the DCO issue, and create a pull request again after the changes so, can you please review it?

**Comment by ShuvamChatterjee28** [comment]
> @ShuvamChatterjee28, I think I fixed the DCO issue, and create a pull request again after the changes so, can you please review it?

It is still failed, please check the DCO label. Have you created a sign-off key first? If so, then again follow the steps.

**Comment by Manishch27** [comment]
@ShuvamChatterjee28 Do i have to commit the changes from master branch? Because i don't see the sign-Off label in master branch.

**Comment by l5io** [comment]
🚀 Preview for commit 9590a8a75afa3c3327dc1bcec044c49fd281aaf3 at: https://63f27771d54a53123a11803e--layer5.netlify.app

File: src/sections/Landscape/LandscapeGrid.style.js (hunk 1 above)
Comment (Nikhil-Ladha) [reply src/sections/Landscape/LandscapeGrid.style.js]: Use the above mentioned colors, please.

**Comment by Manishch27** [comment]
@Nikhil-Ladha and @ShuvamChatterjee28 I tried everything, but the DCO issue doesn't get fixed. please! help to resolve it.

**Comment by Nikhil-Ladha** [comment]
> @Nikhil-Ladha and @ShuvamChatterjee28 I tried everything, but the DCO issue doesn't get fixed. please! help to resolve it.

You can ignore the DCO check, but please update the code as per my last comment [here](https://github.com/layer5io/layer5/pull/3756#discussion_r1111309783)

File: src/sections/Landscape/LandscapeGrid.style.js [hunk 3]
```diff
@@ -149,7 +152,7 @@ export const LandscapePageWrapper = styled.div`
 		margin: 2.5em 0;
 	}
 	.vertical-timeline::before {
-		background: #1E2117;
+		background: ${props => props.theme.text};
```
Comment (Nikhil-Ladha) [inline src/sections/Landscape/LandscapeGrid.style.js:155]: This is not the color I mentioned in my last comment.

File: src/sections/Landscape/LandscapeGrid.style.js [hunk 4]
```diff
@@ -166,7 +169,7 @@ export const LandscapePageWrapper = styled.div`
 	.meshBubble {
 		background: #00B39F;
 		margin-top: 5rem;
-		border: 8px solid #1E2117;
+		border: 8px solid ${props => props.theme.text};
```
Comment (Nikhil-Ladha) [inline src/sections/Landscape/LandscapeGrid.style.js:172]: Same as above

**Comment by l5io** [comment]
🚀 Preview for commit e46c40d6605ce1a334e633ba4072a7a61479b1ac at: https://63fa490632824343f710fe41--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**